src/lib/falcor/routes/articles/by-slug.js

```diff
@@ -42,7 +42,7 @@ export default [
     },
     set: async jsonGraphArg => {
       const articles = jsonGraphArg.articles.bySlug;
-      const flag = await updateArticles(articles);
+      const flag = await updateArticles('slug', articles);
       if (!flag) {
         throw new Error(
           'For unknown reasons updatePostMeta returned a non-true flag',
```

Comment:
In line 54 below, you are setting the value with the value of the field fetched from the updateArticles. This should not be set as a value, but rather as a ref pointing to the article by its id. Therefore, when a set request happens on this route, you should 1) get the id of the article requested, 2) update the article based on its id, 3) update the articles bySlug route given references to the article with its id.

Reply:
This goes for the remaining sets in the file, that are not updating articles based on their id.